trailblazer/utils/datetime.py

```diff
@@ -26,11 +31,17 @@ def get_datetime_from_timestamp(timestamp: str, datetime_formats: List[str]) ->
     for datetime_format in datetime_formats:
         try:
             return datetime.strptime(timestamp, datetime_format)
-        except ValueError:
+        except ValueError as error:
+            LOG.error(f"Error converting timestamp: {error}")
             continue
 
 
 def tower_datetime_converter(timestamp: str) -> datetime:
     """Converts a NF Tower timestamp into a datatime object."""
     allowed_formats = [TOWER_TIMESTAMP_FORMAT, TOWER_TIMESTAMP_FORMAT_ALTERNATIVE]
     return get_datetime_from_timestamp(timestamp, allowed_formats)
+
+
+def get_date_days_ago(days_ago: int) -> datetime:
+    """Return the date that was number of 'days_ago'."""
+    return datetime.now() - timedelta(days=days_ago)
```

Comment:
I think the current function could be clearer with the following suggestion. Just a personal preference.

```suggestion
def get_date_n_days_ago(n: int) -> datetime:
    """Return the date that was number (n) of days_ago."""
    return datetime.now() - timedelta(days=n)
```

Reply:
Changed to `get_date_number_of_days_ago`